packages/validator/src/validator.ts

```diff
@@ -159,6 +162,32 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        let feeRecipient = "";
```

Comment:
setting an empty string here is not ideal, please check if it is possible to use a better default value here

https://github.com/ChainSafe/lodestar/blob/df1ef97e9050a588b74f21bfae314d675caea288/packages/validator/src/services/validatorStore.ts#L149

https://github.com/ChainSafe/lodestar/blob/df1ef97e9050a588b74f21bfae314d675caea288/packages/validator/src/services/validatorStore.ts#L187